src/app/EventLogging.h

```diff
@@ -73,6 +73,8 @@ CHIP_ERROR LogEvent(const T & aEventData, EndpointId aEndpoint, EventNumber & aE
     eventOptions.mPriority    = aEventData.GetPriorityLevel();
     eventOptions.mFabricIndex = aEventData.GetFabricIndex();
 
+    VerifyOrReturnError(eventOptions.mFabricIndex != kUndefinedFabricIndex, CHIP_NO_ERROR);
```

Comment:
Suggest adding a comment that this skips logging the event if it's fabric-scoped but no fabric association exists yet.

Reply:
done, thanks

Reply:
would follow-up it, https://github.com/project-chip/connectedhomeip/issues/21982, thanks